fpsim/sim.py

```diff
@@ -1395,21 +1395,83 @@ def make_people(self, n=1, age=None, sex=None, method=None, debut_age=None):
         if self.regional == True:
             region = self.initialize_region(n)
             urban = self.initialize_urban(n, self['urban_prop'], region)
+            #barriers by region
```

Comment:
Great work in adding all of this logic! Since there is so much here, I would separate each of the parameter initialization to be a separate function (similar to initialize_region and initialize_urban functions) so that this is a bit cleaner and easier to troubleshoot if need be.

Reply:
Thanks! Each parameter initialization is now written as a separate function.